src/plugin.ts

```diff
@@ -125,7 +125,10 @@ export class PluginContainer {
       .map((file): ChunkInfo | AssetInfo => {
         if (isJS(file.path) || isCSS(file.path)) {
           const relativePath = path.relative(process.cwd(), file.path)
-          const meta = metafile?.outputs[relativePath]
+          const meta =
```

Comment:
seems not relate to this PR

Reply:
it's a bug in the current implementation, this meta data will be used at https://github.com/egoist/tsup/blob/b749a274be69aab14c64bc926a60c84faef7d173/src/plugins/cjs-interop.ts#L15


this test case will pass on a unix system but fail on windows without this change:
```ts
import { runInNewContext } from 'vm'

test.only('cjsInterop', async () => {
  async function runCjsInteropTest(
    name: string,
    files: Record<string, string>
  ) {
    const { output } = await run(`${getTestName()}-${name}`, files, {
      flags: ['--format', 'cjs', '--cjsInterop', '--splitting'],
    })
    const exp = {}
    const mod = { exports: exp }
    runInNewContext(output, { module: mod, exports: exp })
    return mod.exports
  }

  await expect(
    runCjsInteropTest('simple', {
      'input.ts': `export default { hello: 'world' }`,
    })
  ).resolves.toEqual({ hello: 'world' })
})
```

Reply:
Can we reuse the function?
https://github.com/egoist/tsup/blob/83c7c7f3131ca6d46aaad4de3111c2fd5e5b5c21/src/utils.ts#L113